src/main/java/com/vdovich/tbfm/bot/Bot.java

```diff
@@ -0,0 +1,110 @@
+package com.vdovich.tbfm.bot;/* Created by user on 03.01.20 */
+
+import com.vdovich.tbfm.service.INasaService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.TelegramBotsApi;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.methods.send.SendPhoto;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.InlineKeyboardMarkup;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.InlineKeyboardButton;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.PostConstruct;
+
+@Component
+public class Bot extends TelegramLongPollingBot {
+    private static String BOT_NAME = "SortingBot";
+    private static String API_TOKEN = "908897882:AAHGta7UI5RmcP6w9di-vX465BIne0iTKIo";
+    
+    @Autowired
+    INasaService service;
+    
+    @PostConstruct
+    public void registerBot(){
+        TelegramBotsApi telegramBotsApi = new TelegramBotsApi();
+        try {
+            telegramBotsApi.registerBot(this);
+        } catch (TelegramApiException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public SendMessage sendInlineKeyboard(long chatId) {
+        SendPhoto photo = new SendPhoto().setChatId(chatId).setPhoto("https://telegram-bot-sdk.readme.io/reference#forwardmessage");
+        InlineKeyboardMarkup inlineKeyboardMarkup = new InlineKeyboardMarkup();
+
+        List<InlineKeyboardButton> keyboardButtonsRow1 = new ArrayList<InlineKeyboardButton>() {{
+            add(new InlineKeyboardButton().setText("Picture of the day").setCallbackData("1"));
```

Comment:
Maybe it would be better to use some static or enum fields for 
```
private static final String PICTURE_OF_THE_DAY = "picture_of_the_day"; //also could be enum property
...
.setCallbackData(PICTURE_OF_THE_DAY)
```
and when getting message change the condition to  `update.getCallbackQuery().getData().equals(PICTURE_OF_THE_DAY)`